compiler/damlc/tests/daml-test-files/InterfaceDesugared.daml

```diff
@@ -7,84 +7,204 @@
 
 -- | The "desugared" counterpart to Interface.daml
 -- Please keep in sync with Interface.daml
+-- You can compare this file with the output of
+-- > bazel run damlc -- compile $REPO/compiler/damlc/tests/daml-test-files/Interface.daml --target=1.dev --ghc-option=-ddump-parsed > desugared.out
```

Comment:
Definitely not for this PR:
Somehow this seems to be getting out of hand. I don’t have a fantastic idea for what to do about it though. At least at this stage it seems very useful to be able to iterate on the desugaring quickly so this is helpful but keeping it up2date is super annoying and it’s becoming so large that it’s hard to maintain.
Maybe we can just use the simplest interface we can possibly come up with?

wdyt @sofiafaro-da @akrmn?